ios/FluentUI/List/ListItem.swift

```diff
@@ -0,0 +1,154 @@
+                }
+            }
+        }
+
+        @ViewBuilder
+        var accessoryView: some View {
+            if let accessoryType = state.accessoryType,
+               let icon = accessoryType.icon,
+               let iconColor = accessoryType.iconColor(tokenSet: tokenSet, fluentTheme: fluentTheme) {
+                let image = Image(uiImage: icon)
+                    .foregroundColor(Color(uiColor: iconColor))
+                if accessoryType == .detailButton {
+                    SwiftUI.Button {
+                        if let onAccessoryTapped = state.onAccessoryTapped {
+                            onAccessoryTapped()
+                        }
+                    } label: {
+                        image
+                    }
+                } else {
+                    image
+                }
+            }
+        }
+
+        @ViewBuilder
+        var backgroundView: some View {
+            if let backgroundColor = state.backgroundStyleType.defaultColor(tokenSet: tokenSet) {
+                Color(backgroundColor)
+            }
+            EmptyView()
+        }
+
+        @ViewBuilder
+        var contentView: some View {
+            HStack(alignment: .center) {
+                leadingView()
+                    .padding(.trailing, ListItemTokenSet.horizontalSpacing)
+                labelsView
+                    .padding(.trailing, ListItemTokenSet.horizontalSpacing)
+                Spacer()
+                trailingView()
+                    .tint(Color(fluentTheme.color(.brandForeground1)))
+                accessoryView
+                    .padding(.leading, ListItemTokenSet.horizontalSpacing)
+            }
+            .padding(EdgeInsets(top: ListItemTokenSet.paddingVertical + 1,
+                                leading: ListItemTokenSet.paddingLeading,
+                                bottom: ListItemTokenSet.paddingVertical + 1,
+                                trailing: ListItemTokenSet.paddingTrailing))
+            .frame(minHeight: minHeight)
+            .background(backgroundView)
+            .listRowInsets(EdgeInsets())
+        }
+        
+        return contentView
+    }
+
+    var state: ListItemState = ListItemState()
+    
+    private var minHeight: CGFloat {
+        if !subtitle.isEmpty {
+            if !footer.isEmpty {
+                return ListItemTokenSet.threeLineMinHeight
+            } else {
+                return ListItemTokenSet.twoLineMinHeight
+            }
+        }
+        
+        return ListItemTokenSet.oneLineMinHeight
+    }
+    
+    @Environment(\.fluentTheme) private var fluentTheme: FluentTheme
+    
+    @ViewBuilder private let leadingView: () -> LeadingView
+    @ViewBuilder private let trailingView: () -> TrailingView
+    
+    private let footer: Footer
+    private let subtitle: Subtitle
+    private let title: Title
+    private let tokenSet: ListItemTokenSet = ListItemTokenSet(customViewSize: { MSFTableViewCellCustomViewSize.zero })
```

Comment:
Suggestions from anyone on if I should also do a typealias for MSFTableViewCellCustomViewSize to be `ListItemCustomViewSize`? 

 Maybe something to worry about in another iteration when more of the custom view properties are added.

Reply:
I think you should  as it'd make it more consistent.